test-cases.nix

```diff
@@ -48,9 +59,22 @@ rec {
       ${pkgs.tree}/bin/tree --version
     fi
     
-    # Test variable expansion
+    # Test variable expansion and loops
     for file in *.nix; do
       echo "Processing $file"
     done
   '';
+  
+  # Test case 4: More generic bash patterns
+  otherScript = ''
+    echo "Generic bash detection test"
+    
+    if [ "$1" = "test" ]; then
+      echo "Test mode"
+    fi
+    
+    for i in {1..5}; do
+      echo "Iteration $i"
+    done
+  '';
```

Comment:
>  otherScript = ''
>     echo "Generic bash detection test"
>     
>     if [ "$1" = "test" ]; then
>       echo "Test mode"
>     fi
>     
>     for i in {1..5}; do
>       echo "Iteration $i"
>     done
>   '';

This section works with this PR's version of nvim. This is gret the bash highlighting works well.